docs/container.md

```diff
@@ -4,9 +4,11 @@
 
 Nextflow supports a variety of container runtimes. Containerization allows you to write self-contained and truly reproducible computational pipelines, by packaging the binary dependencies of a script into a standard and portable format that can be executed on any platform that supports a container runtime. Furthermore, the same pipeline can be transparently executed with any of the supported container runtimes, depending on which runtimes are available in the target compute environment.
 
-:::{note}
-When creating your container image to use with Nextflow, make sure that Bash (3.x or later) and `ps` are installed in your image, along with other tools required for collecting metrics (See  {ref}`this section <execution-report-tasks>`). Also, Bash should be available on the path `/bin/bash` and it should be the container entrypoint.
-:::
+## Requirements
+
+When creating a container image to use with Nextflow, make sure that Bash (3.x or later) and `ps` are installed in the image, along with other tools required for collecting metrics (See {ref}`this section <execution-report-tasks>`). Bash should be available on the path `/bin/bash` and it should be the container entrypoint.
+
+When using AWS Batch, containers must be able to use the {ref}`AWS CLI <aws-batch-cli>` (unless you are using {ref}`wave-page` and {ref}`fusion-page`).
```

Comment:
Don't think this should be mentioned here. I'd replace with a tip about Wave e.g. "Consider using Wave containers to streamline the provisioning containers on-demand for your data pipeline"